scripts/run-travis-tests.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -ev
+npm run lint
+npm run test:coverage
+npm run test:karma
+cat ./test/coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js
+if git log ${TRAVIS_COMMIT_RANGE} | grep -Ei '\[codemod\]'; then
```

Comment:
I saw this ruby example with a different conditional in the travis docs which inspired it: https://docs.travis-ci.com/user/customizing-the-build/#Implementing-Complex-Build-Steps 

Let me know if you want me to make the restriction looser. I was wondering if including the brackets is too strict. Should we just check for codemod without the brackets?